chapter_tensor_program/tensorir_exercises.md

```diff
@@ -273,7 +273,7 @@ b, i, j, k = sch.get_loops(Y)
 # Step 3. Organize the loops
 k0, k1 = sch.split(k, ...)
 sch.reorder(...)
-sch.compute_at(...)
+sch.reverse_compute_at(...)
```

Comment:
`compute_at` also works in this case. Please revert this line or we can say `sch.compute_at/reverse_compute_at(...)`